samples/sensor/apds9960/README.rst

```diff
@@ -1,52 +1,36 @@
-.. _apds9960-sample:
+.. _apds9960:
 
-RGB and Gesture Sensor
-######################
+APDS9960 RGB, ambient, Gesture Sensor
```

Comment:
According to datasheet https://cdn.sparkfun.com/assets/learn_tutorials/3/2/1/Avago-APDS-9960-datasheet.pdf

All three things has been mentioned and there for data registers to fetch data one for ambient light mean white light brightness and another three for RGB for IMHO this title provides more info than you suggested